psamm/datasource/sbml.py

```diff
@@ -648,6 +649,44 @@ def __init__(self):
         self._namespace = SBML_NS_L3_V1_CORE
         self._sbml_tag = partial(_tag, namespace=self._namespace)
 
+    def _make_safe_id(self, thisid):
+        """Makes sure the ID only has the allowed characters.
```

Comment:
Usually we should prefer be explicit about whether the function returns a value and what it returns, e.g.: "Return a modified id that has been made safe for SBML". Also, call the parameter `id`.